attest/attest.go

```diff
@@ -312,6 +312,30 @@ func (a HashAlg) String() string {
 	return fmt.Sprintf("HashAlg<%d>", int(a))
 }
 
+// PlatformAttestation encapsulates the set of information necessary to attest
+// the booted state of the machine the TPM is attached to.
+//
+// The digests contained in the event log can be considered authentic if:
+//  - The AIK public corresponds to the known AIK for that platform.
+//  - All quotes are verified with AIKPublic.Verify(), and return no errors.
+//  - The event log parsed successfully using ParseEventLog(), and a call
+//    to EventLog.Verify() with the full set of PCRs returned no error.
+type PlatformAttestation struct {
```

Comment:
So we've currently got "ActivationParameters" and "AttestationParameters"

For consistency, can this be "PlatformParameters"?

Reply:
Done.